python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -829,6 +834,9 @@ def _create_vars_from_blocklist(self,
             if not block_map.has_key(varname):
                 block_map[varname] = []
             block_map[varname].append((long(offset), long(size)))
+        # Do not remove this important debug message:
+        print("block map: %s" % block_map)
```

Comment:
need to delete ```print```

Reply:
I intentionally leave this debug message here, this message is very important for debugging, and for distributed training, people may not be bothered by this single log.